packages/manager/modules/exchange/src/account/update/update.routing.js

```diff
@@ -0,0 +1,26 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('exchange.dashboard.account.update', {
+    url: '/update',
+    component: 'exchangeAccountUpdate',
+    params: {
+      emailAccount: null,
+    },
+    redirectTo: (transition) =>
+      Promise.all([transition.injector().getAsync('emailAccount')]).then(
+        ([emailAccount]) => {
+          return !emailAccount ? 'exchange.dashboard.account' : false;
+        },
+      ),
+    resolve: {
+      breadcrumb: /* @ngInject */ ($translate) =>
+        $translate.instant('exchange_account_update'),
+
+      goBack: /* @ngInject */ ($state) => () => $state.go('^'),
+
+      emailAccount: /* @ngInject */ ($transition$) => {
+        console.log('ZM:: params', $transition$.params());
```

Comment:
```suggestion
```

Reply:
Removed, thanks.

Reply:
Forgot to update ? We can still see it

Reply:
Yes, i have not yet ask for a re-review.
I just pushed changes.

Thanks